cmd/skywire-cli/commands/config/gen.go

```diff
@@ -86,34 +87,39 @@ var genConfigCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, _ []string) {
 		mLog := logging.NewMasterLogger()
 		mLog.SetLevel(logrus.InfoLevel)
+		var services visorconfig.Services
 
 		urlstr := []string{"http://", serviceConfURL, "/config"}
```

Comment:
We need an if statement here for the `-t` flag. If it is set we need to call `/config/dev` to get dev urls.